content/guide/ui/styling.md

```diff
@@ -0,0 +1,694 @@
+| `tab-background-color`                 | `tabBackgroundColor`               | Both        | `TabView`     | Sets the background color of the tabs.                                                                                           |
+| `tab-text-font-size`                   | `tabTextFontSize`                  | Both        | `TabView`     | Sets the tab titles font size, without changing the font size of all contents of the tab.                                        |
+| `text-transform`                       | `textTransform`                    | Both        | `TabViewItem` | Sets the text transform individually for every `TabViewItem`. Value options: `capitalize`, `lowercase`, `none`, and `uppercase`. |
+| `android-selected-tab-highlight-color` | `androidSelectedTabHighlightColor` | **Android** | `TabView`     | Sets the underline color of the tabs in Android.                                                                                 |
+| `android-elevation`                    | `androidElevation`                 | **Android** | `View`        | Sets the elevation of the View in Android.                                                                                       |
+| `android-dynamic-elevation-offset `    | `androidDynamicElevationOffset`    | **Android** | `View`        | Sets the elevation of the View in Android, which will be shown when an action was performed(e.g. `tap`, `touch`).                |
+| `off-background-color`                 | `offBackgroundColor`               | Both        | `Switch`      | Sets the background color of the Switch when it is turned off.                                                                   |
+| `highlight-color`                      | `highlightColor`                   | Both        | `TabStrip`    | Gets or sets the underline color of the selected `TabStripItem`.                                                                 |
+
+::: warning Note
+Currently, we can set only the `backgroundColor`, `color`, `fontFamily`, `fontSize`, `fontStyle`, `fontWeight` and `textTransform` styling properties to the `Label` and `Image` components inside the TabStripItem. More about the usage of those properties can be found in the [Supported CSS Properties](#supported-css-properties) section.
+:::
+
+::: warning Note
+On iOS, the TabStripItems can not be stylied individually.
+:::
+
+### Adding a shadow to a View on Android
+
+On Android, to add a shadow to a View instance use the `androidElevation` property.
+
+```xml
+<StackLayout class="home-panel">
+  <TextView
+    class="tvElevation"
+    editable="false"
+    textWrap="true"
+    text="TextView"
+  ></TextView>
+  <label androidElevation="5" class="sampleLabel" textWrap="true" text="Label"></label>
```

Comment:
Are the tags for `label` and `button` right?  Do they need to start with an uppercase character, or is it entirely optional?